src/isar_exr/robotinterface.py

```diff
@@ -155,21 +163,27 @@ def step_status(self) -> StepStatus:
     def stop(self) -> None:
         try:
             self.api.pause_current_mission(self.exr_robot_id)
-        except RobotException:
-            raise
-        except Exception as e:
-            raise RobotException from e
+        except Exception:
+            message: str = "Could not stop the running mission"
+            self.logger.error(message)
+            raise RobotException(
+                error_reason=ErrorReason.RobotCommunicationException,
+                error_description=message,
+            )
 
     def get_inspections(self, step: InspectionStep) -> Sequence[Inspection]:
         raise NotImplementedError
 
     def initialize(self, params: InitializeParams) -> None:
         try:
             self.api.wake_up_robot(self.exr_robot_id)
-        except RobotException:
-            raise
-        except Exception as e:
-            raise RobotException from e
+        except Exception:
+            message: str = "Could not initialize mission"
```

Comment:
This should be could not initialize robot instead of mission, right?

Reply:
Yeah, done =)